### DISCUSSION TITLE
Copilot Service Restored 👍  We Apologize for the Inconvenience

### DISCUSSION THREAD

**Discussion by iamcjwilliams** [community/community#44262] [Copilot Conversations]
### Product Feedback

The Copilot team would like to express its regret for the recent modifications to our rate limits, which caused some Copilot users to encounter some discomfort. We understand that these changes may have impacted your ability to fully enjoy Copilot and we apologize for any frustration this may have caused.

We are happy to announce that the issue has been resolved and service has been restored to normal! Our team has worked hard to fix the problem and ensure that everyone can enjoy Copilot without interruption!

We appreciate your continued support of Copilot. We're excited to keep providing outstanding service to you and maintaining our dedication to a positive customer experience. 🚀

**Comment by ettaboyle** [comment]
### GitHub Community Note
We are pinning this post to the Community Discussions front page Jan 13-16 for user awareness. 
Comments have been disabled but users can still react with emojis.

Thank you! ♥️
